articles/aks/learn/quick-windows-container-deploy-cli.md

```diff
@@ -27,6 +27,8 @@ This article assumes a basic understanding of Kubernetes concepts. For more info
 - If you have multiple Azure subscriptions, select the appropriate subscription ID in which the resources should be billed using the
 [az account](/cli/azure/account) command.
 
+- The Azure subscription you're using must be registered to use namespace `Microsoft.OperationsManagement`. To check this, follow the instructions in the "Solution" section of [Resolve errors for resource provider registration](https://learn.microsoft.com/azure/azure-resource-manager/troubleshooting/error-register-resource-provider?tabs=azure-portal#solution).
```

Comment:
To maintain consistency with our other articles and the prerequisite section, please replace this proposed change with the following:

- Verify *Microsoft.OperationsManagement* and *Microsoft.OperationalInsights* providers are registered on your subscription. These are Azure resource providers required to support [Container insights][azure-monitor-containers]. To check the registration status, run the following commands:

    ```azurecli
    az provider show -n Microsoft.OperationsManagement -o table
    az provider show -n Microsoft.OperationalInsights -o table
    ```

    If they are not registered, register *Microsoft.OperationsManagement* and *Microsoft.OperationalInsights* using the following commands:

    ```azurecli
    az provider register --namespace Microsoft.OperationsManagement
    az provider register --namespace Microsoft.OperationalInsights
    ```

> [!NOTE]
> Run the commands with administrative privileges if you plan to run the commands in this quickstart locally instead of in Azure Cloud Shell.

At the end of the article, after line 412 add the following to resolve the link to Azure Monitor Container insights:

[azure-monitor-containers]: ../azure-monitor/containers/container-insights-overview.md

Reply:
I believe your comment is missing a link? Otherwise, sounds good!

Reply:
@EliiseS - Opps, sorry. Here is the link ref for line 412:

[azure-monitor-containers]: ../../azure-monitor/containers/container-insights-overview.md

Reply:
I am now not sure if I'm part of practical joke or not 😄 

You still didn't add the link 😆

Reply:
Aye Caramba! I've added the markdown twice and now seeing that for some reason, it's being misinterpreted and removed. So let me try this - "[azure-monitor-containers]: ../../azure-monitor/containers/container-insights-overview.md" Remove the double-quotes when pasting.

Reply:
@EliiseS - Sorry, I wasn't making a joke at your expense. The markdown snippit was getting misinterpreted in my response so I've wrapped it around double-quotes to make sure it shows up.

Reply:
@MGoedtel It's really fine, no problem 😆 I figured you were having some problems somewhere in the pasting process

Reply:
I am a bit confused what the note about administrative privileges means? How does it make a difference to run the  `az` commands with sudo or not? Or does administrative privileges mean something else here?